app/helpers/alerts_helper.rb

```diff
@@ -1,3 +1,11 @@
 # frozen_string_literal: true
 module AlertsHelper
+  def alert_display(alert)
+    return '✓' if alert.resolved?
+    resolve_button(alert) if alert.resolvable? # implicit return
```

Comment:
This has three possible results: a check mark, the button, or nothing.

That will be clearer if you make this three lines, with a guard clause before this line: `return unless alert.resolvable?`